tools/topology/topology2/include/common/tokens.conf

```diff
@@ -181,4 +181,8 @@ Object.Base.VendorToken {
 		node_type		1980
 		deep_buffer_dma_ms	1981
 	}
+	"19" {
+		name "process"
```

Comment:
seems currently, you only want this used for RTC-AEC, is it possible to make it as a common attribute?
CPC calculation need set different input/output buffer size, so maybe we can start from here.

Reply:
Rander suggested to hide IBS/OBS adjustments inside the tplg instead of asking driver to run the math, which is fine.